src/lib/overlay-language-service-host.ts

```diff
@@ -81,6 +82,30 @@ export class OverlayLanguageServiceHost implements ts.LanguageServiceHost {
     return contents;
   }
 
+  resolveModuleNames(
+    moduleNames: string[],
+    containingFile: string
+  ): ts.ResolvedModule[] {
+    const resolvedModules: ts.ResolvedModule[] = [];
+    for (const moduleName of moduleNames) {
+      let result = ts.resolveModuleName(moduleName, containingFile, this.compilerOptions, {
+        fileExists: this.fileExists.bind(this),
+        readFile: this.readFile.bind(this),
+      });
+      if (result.resolvedModule) {
+        resolvedModules.push(result.resolvedModule);
+      } else {
+        // If default resolution doesn't work, try node resolution.
```

Comment:
When is this necessary? I'd be fine relying on TypeScript's default resolution, or figuring out why it's not working (I believe it does node resolution, at least with the right setting).

Reply:
![giphy (1)](https://user-images.githubusercontent.com/7411489/65191667-9a5fe180-da29-11e9-8ab1-35232b468582.gif)

Reply:
...thanks haha. I'll remove it and flip the `moduleResolution` compiler option to 'Node' 🤣